packages/asc-ui/src/components/Tabs/Tabs.tsx

```diff
@@ -33,7 +33,7 @@ function formatPanelId(id: string) {
   return `panel-${id}`
 }
 
-export const Tabs: React.FC<Tabs> = ({ label, children, ...otherProps }) => {
```

Comment:
Why did you remove `otherProps` here? The idea was that the user can set props (such as `style`) on the `Tabs` component, in order to style `TabList `for example.